docs/intro/icpc.md

```diff
@@ -1,54 +1,54 @@
 author: NachtgeistW
 
-## 赛事介绍
+## Introduction
 
 ### ICPC
 
- **ICPC** （英文：International Collegiate Programming Contest，中文：国际大学生程序设计竞赛）由 ICPC 基金会（英文：ICPC Foundation）举办，是最具影响力的大学生计算机竞赛。由于以前 ACM 赞助这个竞赛，也有很多人习惯叫它 ACM 竞赛。
+ **ICPC**, International Collegiate Programming Contest, is organized by the ICPC Foundation. It is the most influential programming competition for college students. Since ACM sponsored this competition in the past, many people are used to calling it ACM competition.
```

Comment:
```suggestion
International Collegiate Programming Contest, or ICPC, is a competitive programming contest organized by the ICPC Foundation. It is the most influential programming competition for college students. Since ACM sponsored this competition in the past, many people are used to calling it ACM competition.
```